criu/cr-dump.c

```diff
@@ -1963,6 +1963,9 @@ int cr_pre_dump_tasks(pid_t pid)
 	if (collect_pstree())
 		goto err;
 
+	if (checkpoint_devices())
```

Comment:
I don't understand why checkpoint_devices() should be called on pre-dump? How should it work?